site/documentation/content/deployment/create-kubernetes-cluster.md

```diff
@@ -3,150 +3,6 @@ title = "Setting up a Kubernetes Cluster"
 weight = 480
 +++
 
-This guide describes how to set up a Kubernetes cluster which can be used to run Eclipse Hono&trade;.
-
-<!--more-->
-
-Hono can be deployed to any Kubernetes cluster running version 1.11 or newer. This includes [OpenShift (Origin)](https://www.okd.io/) which is built on top of Kubernetes.
-
-The [Kubernetes web site](https://kubernetes.io/docs/setup/) provides instructions for setting up a (local) cluster on bare metal and/or virtual infrastructure from scratch or for provisioning a managed Kubernetes cluster from one of the popular cloud providers.
-
-<a name="Local Development"></a>
-
-## Setting up a local Development Environment
```

Comment:
There are still links to this chapter in the Getting Started guide (relref `/deployment/create-kubernetes-cluster#setting-up-a-local-development-environment`).